src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -80,6 +81,15 @@ class AddSecondaryLoginPage extends Component {
         Onyx.merge(ONYXKEYS.USER, {error: ''});
     }
 
+    onSecondaryLoginChange(login) {
+        if (this.formType === CONST.LOGIN_TYPE.EMAIL) {
+            this.setState({login});
+        } else if (
+            this.formType === CONST.LOGIN_TYPE.PHONE && (CONST.REGEX.DIGITS_AND_PLUS.test(login) || login === '')) {
```

Comment:
The style in this condition looks odd, I would make a new line on the `&&` instead of moving the whole condition to a new line while leaving the parenthesis behind.

Reply:
Updated.